server/src/main/java/com/cloud/api/query/dao/UserVmJoinDaoImpl.java

```diff
@@ -78,6 +81,8 @@
     private UserDao _userDao;
     @Inject
     private NicExtraDhcpOptionDao _nicExtraDhcpOptionDao;
+    @Inject
+    UserStatisticsDao _userStatsDao;
```

Comment:
I don't want to sound repetitive and I would not block a PR due to `_`, in any circumstance. But I would really appreciate it if you could remove `_` from any added variable :slightly_smiling_face: